hawkular-alerts-rest/hawkular-alerts-rest-api/src/main/java/org/hawkular/alerts/rest/ActionPluginHandler.java

```diff
@@ -70,18 +70,17 @@ public ActionPluginHandler() {
             response = String.class, responseContainer = "List")
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "Successfully fetched list of actions plugins."),
+            @ApiResponse(code = 400, message = "Bad Request/Invalid Parameters.", response = ApiError.class),
```

Comment:
I opted for consistency and added the 400 as a possible response given the new handler, even if at the current time we know the internals won't return IllegalArgumentException.  But by just listing 400 as a possibility it means the impl below the handler is free to return IllegalArgumentException without us having to change anything in the handler.  If you're totally opposed I can remove the 400 annotations, but I personally don't think it's an issue.